framework/include/fvkernels/FVScalarLagrangeMultiplier.h

```diff
@@ -16,8 +16,12 @@
  *
  * \int \phi = \int \phi_0
  *
- * using a Lagrange multiplier approach. E.g. this kernel enforces the constraint that the average
- * value of \phi matches \phi_0
+ * or
+ *
+ * \phi(r_0) = \phi_0
+ *
+ * using a Lagrange multiplier approach. E.g. this kernel enforces the constraint that
+ * the average or point value of \phi matches \phi_0
```

Comment:
imo we need to split things here.
FVScalarLagrangeMultiplier should be the base class for a bunch of constraints:
- integral value
- point value
- no backward flow 
- etc

Reply:
I think this is in your PR so I'll just mark this guy resolved.